apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2120,14 +2140,20 @@ static void libc_stdio_meminstream_tc(void)
 */
 static void libc_stdio_memoutstream_tc(void)
 {
-	FAR char buf[STDIO_BUFLEN];
+	char *buf = STREAM_TEST_CONTENTS;
 
 	struct lib_memoutstream_s memoutstream;
 
+	/* Check with valid BUFLEN */
+
 	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buf, STDIO_BUFLEN);
 	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buf));
 	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buflen, (STDIO_BUFLEN - 1));	/* Save space for null terminator, hence checing with (STDIO_BUFLEN-1)*/
 
+	memoutstream.public.put((FAR struct lib_outstream_s *)&memoutstream.public, buf[0]);
```

Comment:
Are we sure, buf is not NULL here?

Reply:
yes , after above assignment (line 2143) this value doesn't change.